manager/state/raft/raft.go

```diff
@@ -31,6 +31,11 @@ import (
 )
 
 var (
+	// ErrPingFailure is returned when there is an issue with the initial handshake which means
+	// that the address provided must be invalid or there is ongoing connectivity issues at join time.
+	ErrPingFailure = errors.New("raft: cannot ping potential new raft member")
```

Comment:
How about `raft: could not connect to prospective new cluster member using its advertised address`